resources/public/api/conf/1.0/examples/create-organisation-response.json

```diff
@@ -33,5 +33,6 @@
   "pensionSchemeAdministratorIdentifier":"A1234567",
   "eoriNumber":"GB123456789012",
   "crn": "12345678",
+  "personPresentingTheGoods": "GB123456789012",
```

Comment:
This should not be part of this PR - it fits better with https://github.com/hmrc/api-platform-test-user/pull/177